### PR TITLE
commit homework 5.2

### PR DESCRIPTION
**Pull request by AnkaMikola**

### CODE REVIEW

File: Homeworks/homework_05.2.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    ('Samuel', 'Jones', 50, 'Business Consultant', 'Atlanta'),
+    ('Emma', 'Hall', 31, 'Chef', 'Dallas'),
+    ('William', 'Clark', 29, 'Financial Analyst', 'Houston'),
+    ('Ava', 'White', 42, 'Journalist', 'San Diego'),
+    ('Ethan', 'Anderson', 36, 'Product Manager', 'Phoenix'),
+]
+print('\n-----Task 1-----\n')
+
+addition = ('Vicky', 32, 'Lawyer', 'Zurich')
+people_records.extend(addition)
```
Comment (antohaUa) [inline Homeworks/homework_05.2.py:28]: Incorrect cause you need to add record to the beginning

File: Homeworks/homework_05.2.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+people_records[1], people_records[5] = people_records[5], people_records[1]
+print(people_records)
+
+print('\n-----Task 3-----\n')
+
+ages = []
+ages.append(people_records[6][2])
+ages.append(people_records[10][2])
+ages.append(people_records[13][2])
```
Comment (antohaUa) [inline Homeworks/homework_05.2.py:41]: Please try to organize this as list comprehension

File: Homeworks/homework_05.2.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+all_above_30 = all(age >= 30 for age in ages)
```
Comment (antohaUa) [inline Homeworks/homework_05.2.py:43]: 30 better to move to the top as constant

**Review by antohaUa — COMMENTED** [review]
Code should be fixed

**Comment by AnkaMikola** [comment]
https://github.com/AnkaMikola/AQA_Hillel_git/pull/23

**Pull request merged**